blueoil/datasets/ilsvrc_2012.py

```diff
@@ -57,8 +56,8 @@ def __init__(
     @functools.lru_cache(maxsize=None)
     def classes(self):
         # wget https://raw.githubusercontent.com/Lasagne/Recipes/master/examples/resnet50/imagenet_classes.txt
-        df = pd.read_csv(os.path.join(self.data_dir, 'imagenet_classes.txt'), sep="\n", header=None)
-        return df[0].tolist()
+        linelist = [line.rstrip('\n') for line in open(os.path.join(self.data_dir, 'imagenet_classes.txt'))]
```

Comment:
Can we use `readlines` instead of `line.rstrip('\n')` to reduce code?

Reply:
If we want to use readlines, we need to write the codes as follow, which will have more codes than the one written above.
```
with open(os.path.join(self.data_dir, 'imagenet_classes.txt'), "r") as f:
    linelist = f.readlines()
```
And this will include the unwanted '/n' in the list.

Reply:
I see.